ext/rbkit_event.c

```diff
@@ -1,6 +1,6 @@
 #include "rbkit_event.h"
 
-static size_t correlation_id = 0;
+static int correlation_id = 0;
```

Comment:
Max value of `correlation_id` = the total number of object space dumps i made after running the server. I thought int would be better. Well, we can keep size_t if you think it's better, not a biggie.